releases/handbook.md

```diff
@@ -227,6 +227,7 @@ From that phase and forward updates to the manifests repo must only be fixing co
 - Create a new **vX.Y-rc.0** tag that will be pointing to the branch created above
 - Add commits on top of the **vX.Y-branch** branch to include bug fixes. The other WGs can provide a new _git revision_ to Manifests WG leads. The new git revision must only include bug fixes, not new features.
 - On the last day of feature freeze, cut a new **vX.Y-rc.1** RC tag on the **vX.Y-branch** release branch.
+- Note - The Notebooks Working Group's versioning policy follows the Manifests Working Group's process.
```

Comment:
@annajung You make a good point.   My observation is that we could be duplicating the same policy in 3 places i.e. the release handbook, the notebooks WG and the manifest WG.     Would it make sense to point all references to one place?

Reply:
I don't think you have to duplicate the policy, just need to add a pointer to the release handbook with the note you add here but in the Notebook repo README or another appropriate doc in the notebook repo. Right now, there is no doc in the notebook repo that helps them to navigate back to this doc.

Reply:
@annajung @kimwnasptd  I looked at the Notebook repo.   It appears to me that Versioning could fit under Charter.    I modified the charter to include a versioning policy.   For consistency, I referred the details of the NB policy back to the Manifest versioning policy in the release handbook, https://github.com/kubeflow/community/pull/603.    Does that satisfy the suggestion above ?

Reply:
I also created an .md file for the Manifest repo, which also links to the Kubeflow Release Handbook, https://github.com/kubeflow/manifests/pull/2373

Reply:
I think many users will look for the versioning info in the `kubeflow/kubeflow` repo and may not navigate to the charter page which is under `kubeflow/community`. Therefore, I think it might be better to keep it consistent with other WG and have the versioning policy in the corresponding repo. Since how to structure the docs is up to the WG leads, @kimwnasptd can chime in here and decide where would be the most appropriate place.